Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -872,58 +872,47 @@ public Task Sort( DataGridColumn<TItem> column, SortDirection? sortDirection = n
     /// If more than one column is specified, <see cref="SortMode"/> must be <see cref="DataGridSortMode.Multiple"/>
     /// </remarks>
     public async Task ApplySorting(params DataGridSortColumn[] columns ) {
-        if ( columns.Length > 1 && SortMode == DataGridSortMode.Single )
-        {
-            throw new ArgumentException(
-                $"Cannot sort on multiple columns when {nameof(SortMode)} is {nameof(DataGridSortMode.Single)}." );
-        }

```

Comment:
I think it would be best if we just exit the function if not columns have been provided, what do you think?

Reply:
Or upon further though, if you call `ApplySorting` with an empty collection, then no sorting is applied and any sorting is removed, I think that would be expected? What do you think?

Reply:
I was talking to my colleague about what should happen if no columns are passed for sorting (empty array). We both agreed that in this case the "default sorting" of the data source should be active again. Otherwise a public "ResetSorting" method would be required.

Reply:
I'm sorry I am still not convinced. you explicitly pass in an empty array.
I would expect the behaviour of calling ApplySorting() and ApplySorting(Array.Empty) to be different.
(Granted we don't have the version without an argument)

On the first option I could expect the behaviour you guys are mentioning, on the second, I'm explicitly passing in the collection I want to sort on, which is empty, therefore I will sort on no columns, which is basically an "empty" sorting, or no sorting at all.

That's my thought process, what do you think?
@stsrki you have any thoughts on this?